src/components/Table/TableFormatters/AccountConnectFormatter.vue

```diff
@@ -111,7 +111,9 @@ export default {
       try {
         const url = this.formatterArgs.assetUrl.replace('{id}', assetId)
         const res = await this.$axios.get(url)
-        this.protocols = res.protocols || []
+
+        // TODO 暂时将 sftp 隐藏
+        this.protocols = res.protocols.filter(protocol => (protocol.name !== 'sftp') && (protocol.name !== 'winrm')) || []
       } catch (e) {
         throw new Error(`Error getting protocols: ${e}`)
       }
```

Comment:
There are no differences or potential issues found with the given code snippet up to 2023. However, if you're referring to code from before the knowledge cutoff of September 1st, 2021, please ensure that the changes made since then have been implemented correctly according to modern JavaScript programming practices and standards such as ES6 or newer versions of JavaScript.